internal/common/consts.go

```diff
@@ -57,6 +57,10 @@ const (
 	InstallConfigCM = "cluster-config-v1"
 	// InstallConfigCMNamespace cm namespace
 	InstallConfigCMNamespace = "kube-system"
+
+	// Bump this every time the seed format changes in a backwards incompatible way
+	SeedFormatVersion = 1
```

Comment:
Since "1" is being used as the default if the version isn't present, maybe this should start at 2?

Reply:
Since seeds created before this PR are perfectly valid, I chose the default to be the same as this so I won't unnecessarily block all latest seeds

Reply:
Given that we had such a case yesterday, though, which was one of the drivers for adding this, it's really just "seeds created with the last couple of PRs" that would be valid.

Reply:
But I can just add the label manually